netbout-web/src/test/java/com/netbout/rest/bout/TkAttachTest.java

```diff
@@ -46,10 +50,55 @@
  * @version $Id$
  * @since 2.15.1
  * @checkstyle ClassDataAbstractionCouplingCheck (500 lines)
+ * @checkstyle MultipleStringLiteralsCheck (500 lines)
```

Comment:
@bdragan let's avoid this - maybe slightly change the strings used in the new test.